### PR TITLE
Bug 1189806 - Fix intermittent failure in test_add_to_an_existing_contact_from_messages.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=133e7c3607f89e5bad508e6e70aaccf685502fbf

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6fc4554cadcd026f3558baa10961836a4b4019b1

File: tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_add_to_an_existing_contact_from_messages.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def test_sms_add_number_to_existing_contact(self):
 
         contacts = Contacts(self.marionette)
         contacts.launch()
+        contacts.wait_for_contacts()
```
Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_add_to_an_existing_contact_from_messages.py]: so this assumes that contacts isn't empty right? if we're going to add it inside .launch() method, we should determine whether we're entering an empty contact screen or pre-filled one.

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_add_to_an_existing_contact_from_messages.py]: No, I was thinking of when someone tries to access the contacts.contact property, actually. If one would do that, I think we can safely assume that you want at least one contact to be there.

**Pull request merged**